### PR TITLE
QA-404 Publish packages into separate directories ${DISTRO}-${RELEASE…

### PR DESCRIPTION
**Pull request by merlin-northern**
…}-${ARCH}.

ChangeLog: title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@oleorhagen I just found out that `.template:publish:s3:apt-repo:` needs updating as well.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -338,25 +338,25 @@ test:acceptance:
     - pip3 install -r requirements.txt
     - python3 -m pytest -v
       --mender-client-version $MENDER_VERSION
-      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/mender-client-deb-version)
+      --mender-client-deb-version $(cat ${CI_PROJECT_DIR}/output/opensource/debian-buster-amd64/mender-client-deb-version)
```
Comment (oleorhagen) [inline .gitlab-ci.yml:341]: S:
Why not make a variable out of this while we're at it?

**Review by oleorhagen — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (merlin-northern) [reply .gitlab-ci.yml]: fixed

**Comment by merlin-northern** [comment]
@oleorhagen: acceptance tests alignment is new: https://github.com/mendersoftware/mender-dist-packages/pull/228/files#diff-7fb11d01efb2e6788562533f26902ed0563b68bfbd51ccecf879ec3ad594f869 and I addressed your comments.

**Review by oleorhagen — APPROVED** [review]

**Pull request merged**